### ISSUE TITLE
recent avfctl won't run on macOS 10.14.6

### ISSUE THREAD

**Issue by dericed** [mipops/dvrescue#348]
```
$ dvrescue --version
DVRescue v.0.20.11.20210421 (MediaInfoLib v.21.03) by MIPoPS.

$ avfctl -list_devices
2021-08-26 11:34:36.204 avfctl[58157:788182] Devices:
2021-08-26 11:34:36.205 avfctl[58157:788182] [0] DV-VCR (Sony HVR-M15U)
2021-08-26 11:34:36.205 avfctl[58157:788182] [1] HDV-VCR (Sony HVR-M15U)
2021-08-26 11:34:36.206 avfctl[58157:788182] [2] DV-VCR (Sony DSR-45)
```

few minutes later
```
$ dvrescue --version
DVRescue v.0.20.11.20210825 (MediaInfoLib v.21.03) by MIPoPS.

$ avfctl -list_devices
2021-08-26 11:35:41.969 avfctl[58193:789084] Devices:
dyld: lazy symbol binding failed: Symbol not found: _objc_opt_class
  Referenced from: /usr/local/bin/avfctl (which was built for Mac OS X 11.0)
  Expected in: /usr/lib/libobjc.A.dylib

dyld: Symbol not found: _objc_opt_class
  Referenced from: /usr/local/bin/avfctl (which was built for Mac OS X 11.0)
  Expected in: /usr/lib/libobjc.A.dylib
```

am running macOS 10.14.6

**Comment by dericed** [comment]
Related to recent tests on 10.14.6
```
./dvrescue10303 device://0 --verbosity 7 -m test10302_v7.dv
dyld: Symbol not found: ___NSDictionary0__struct
  Referenced from: /Users/archivesx/Downloads/./dvrescue10303 (which was built for Mac OS X 11.0)
  Expected in: /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
 in /Users/archivesx/Downloads/./dvrescue10303
Abort trap: 6
```

also (I think with 20211002 build.
```
$ dvrescue --version
dyld: Symbol not found: _objc_opt_class
  Referenced from: /usr/local/bin/dvrescue
  Expected in: /usr/lib/libobjc.A.dylib
 in /usr/local/bin/dvrescue
Abort trap: 6
```

**Comment by dericed** [comment]
Also dvrescue 20211101 on 10.14.6
```
$ dvrescue --version
^[[Adyld: Symbol not found: ___NSDictionary0__struct
  Referenced from: /usr/local/bin/dvrescue
  Expected in: /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
 in /usr/local/bin/dvrescue
Abort trap: 6
```

**Comment by dericed** [comment]
An update on avfctl on macOS 10.14.6

with 0710 installer.
```
x$ avfctl -list_devices
2021-11-02 09:37:25.110 avfctl[15190:227672] Devices:
2021-11-02 09:37:25.111 avfctl[15190:227672] [0] DV-VCR (Sony HVR-M15U)
2021-11-02 09:37:25.112 avfctl[15190:227672] [1] HDV-VCR (Sony HVR-M15U)
2021-11-02 09:37:25.112 avfctl[15190:227672] [2] DV-VCR (Sony DSR-45)
```

with 0717 installer.
```
$ avfctl -list_devices
2021-11-02 09:39:15.392 avfctl[15320:229625] Devices:
dyld: lazy symbol binding failed: Symbol not found: _objc_opt_class
  Referenced from: /usr/local/bin/avfctl (which was built for Mac OS X 11.0)
  Expected in: /usr/lib/libobjc.A.dylib

dyld: Symbol not found: _objc_opt_class
  Referenced from: /usr/local/bin/avfctl (which was built for Mac OS X 11.0)
  Expected in: /usr/lib/libobjc.A.dylib

Abort trap: 6

```

**Comment by dericed** [comment]
Retesting with https://mediaarea.net/download/snapshots/binary/dvrescue/20211102-2/dvrescue_CLI_0.20.11.20211102_Mac.dmg

```
Ingest-17:~ archivesx$ ls -l /usr/local/bin/avfctlc
Ingest-17:~ archivesx$ avfctl -device 2 -cmd rew
2021-11-03 11:35:01.040 avfctl[42667:577014] Speed changed: 0.000000 -> -2.000000
Ingest-17:~ archivesx$ avfctl -device 2 -cmd status
2021-11-03 11:35:04.672 avfctl[42668:577073] Invalid command given.
Ingest-17:~ archivesx$ avfctl -device 2 -status
2021-11-03 11:35:10.569 avfctl[42669:577084] Device [2] DV-VCR (Sony DSR-45) status: rewinding
Ingest-17:~ archivesx$ avfctl -device 2 -cmd rew -foreground
2021-11-03 11:35:29.208 avfctl[42670:577164] Speed changed: 0.000000 -> -2.000000
Segmentation fault: 11
```

segmentation fault with -foreground option

**Comment by dericed** [comment]
I think this was fixed, but also don't have access to a 10.14.6 machine anymore. For anyone finding this, please reopen if an issue.
